src/main/java/com/epam/ta/reportportal/core/project/settings/impl/UpdateProjectSettingsHandlerImpl.java

```diff
@@ -111,17 +111,17 @@ public OperationCompletionRS updatePatternTemplate(Long id, String projectName,
 		PatternTemplate patternTemplate = patternTemplateRepository.findById(id)
 				.orElseThrow(() -> new ReportPortalException(ErrorType.PATTERN_TEMPLATE_NOT_FOUND_IN_PROJECT, id, project.getId()));
 
-		if (!patternTemplate.getName().equalsIgnoreCase(updatePatternTemplateRQ.getName())) {
+		final String name = updatePatternTemplateRQ.getName().trim();
```

Comment:
May be better use something like
final String name = StringUtils.trim(updatePatternTemplateRQ.getName());
To avoid possible NPE.

Reply:
done